src/Command/DumpCommand.php

```diff
@@ -343,6 +354,7 @@ protected function getDumpSettingsDefault()
             ] + [
                 'gdpr-expressions' => null,
                 'gdpr-replacements' => null,
+                'gdpr-replacements-file' => null,
```

Comment:
So it's not obvious, but this is possibly the reason you're rereading the file contents of your gdpr-replacements-file in /src/MysqldumpGdpr.php.

If you remove this line, you can also remove the lines 38-42 in MysqldumpGdpr.php.